### PR TITLE
Update Danish translations

### PR DESCRIPTION
**Pull request by richardolsson**
## Description
This PR updates the latest Danish translations after changes made by volunteer translators.

## Screenshots
None

## Changes
* Adds the most recent danish translations from translate.zetkin.org
  * Fixes lots of errors related to capitalization
  * Fixes some incorrect translations

## Notes to reviewer
None

## Related issues
Undocumented

### CODE REVIEW

File: src/locale/nn.yml [hunk 1]
```diff
@@ -713,7 +713,7 @@ feat:
           preview:
             callassignment_goal: fullføringsgruppa for ringeoppdraget "{queryTitle}"
             callassignment_target: målgruppa for ringeoppdraget  "{queryTitle}"
-            standalone: smartsøket
+            standalone: smartsøket "{queryTitle}"
```
Comment (ziggabyte) [inline src/locale/nn.yml:716]: Thinking about if there could be some way that the translator interface/Lyra could indicate whether a translation should begin with big or small letter, if that could be helpful.

File: src/tools/lint-translations.ts [hunk 2]
```diff
@@ -41,14 +41,59 @@ async function run() {
...
+          if (!localArgs[arg]) {
+            error(
+              `Message ${id} of locale ${lang} is not using argument ${arg}`
+            );
+          } else if (!enArgs[arg]) {
+            error(
+              `Message ${id} of locale ${lang} is using unknown argument ${arg}`
+            );
+          }
+        });
```
Comment (ziggabyte) [inline src/tools/lint-translations.ts:96]: would it be possible here to test for correct formatting - big letter/small letter in the beginning of word? or would that better be left to some other part of the interface?

**Review by ziggabyte — APPROVED** [review]
Thought about the things you spoke about, the translations having the wrong format, and made comments about ideas that I had around that. Other than that I see nothing that would be wrong or "smells bad" :)

**Comment by richardolsson** [comment]
Superseded by #1525

File: src/tools/lint-translations.ts (hunk 2 above)
Comment (richardolsson) [reply src/tools/lint-translations.ts:96]: I think that's a great idea for Lyra!

**Pull request closed without merging**